builds/e2e/nested-e2e.yaml

```diff
@@ -0,0 +1,46 @@
+trigger: none
+pr: none
+
+resources:
+  pipelines:
+  - pipeline: images
+    source: 'Azure-IoT-Edge-Core Build Images'
+    branch: master
+    trigger:
+      branches:
+      - master
+  - pipeline: packages
+    source: 'Azure-IoT-Edge-Core Edgelet Packages'
+    branch: master
+    trigger:
+      branches:
+      - master
```

Comment:
It is more flexile if it is defined as pipeline variable with default set to master.